src/types/index.ts

```diff
@@ -81,7 +82,10 @@ declare global {
     // END TODO remove
 
     roamAlphaAPI: {
-      q: (query: string, ...params: unknown[]) => unknown[][];
+      q: <T extends GenericQueryResult>(
```

Comment:
While queries using `:keys` return record/object rows, most queries do not.  Therefore, I believe `q<T extends unknown[] | Record<string, unknown>`> is too permissive.  I'd prefer separate APIs or an explicit opt-in for keyed queries.